src/main/java/mate/academy/spring/model/dto/request/UserRegistrationDto.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.model.dto.request;
+
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+public class UserRegistrationDto {
+    @Email
```

Comment:
You have created your own custom email validator, but you are using the one that javax.validation gives out of the box. Change import

Reply:
oops